packages/xo-remote-parser/src/index.js

```diff
@@ -17,18 +17,21 @@ const parseOptionList = (optionList = '') => {
   const parsed = queryString.parse(optionList)
   Object.keys(parsed).forEach(key => {
     const val = parsed[key]
-    parsed[key] = JSON.parse(val)
+    // some incorrect values have been saved in users database (introduced by #6270)
+    parsed[key] = val === '' ? false : JSON.parse(val)
```

Comment:
How were the incorrect values introduced?

`queryString.stringify({ foo: undefined }) // 'foo='`?

What an horrible default behavior :disappointed:

Reply:
that is exaclty this `allowUnauthorized=' which fails during parsing

Reply:
Are you sure it was saved in the db and we need to support or did the error prevent it from being stored?

Reply:
on my local xo it was saved in database